core/eolearn/tests/test_core_tasks.py

```diff
@@ -426,19 +424,23 @@ def test_explode_bands(
         assert_array_equal(patch[new_feature], patch[feature][..., bands])
 
 
-def test_extract_bands(patch: EOPatch) -> None:
+def test_extract_bands(eopatch_to_explode: EOPatch) -> None:
```

Comment:
ah, I see it is used here as well
:thinking: 

The name `eopatch_with_many_bands` is kinda stupid, so maybe just keep it as it is